app/modules/sightings/models.py

```diff
@@ -711,9 +717,63 @@ def build_identification_request(
         }
         id_request = id_request | id_config_dict
 
-        log.debug(f'sending message to sage :{id_request}')
+        log.debug(f'{debug_context} Built ID message for sage :{id_request}')
         return id_request
 
+    def send_all_identification(self):
+        sighting_guid = str(self.guid)
+        num_jobs = 0
+        # Once we support multiple IA configs and algorithms, the number of jobs is going to grow....rapidly
+        #  also: once we have > 1 config, some annot-level checks will be redundant (e.g. matching_set) so may
+        #    require a rethink on how these loops are nested
+        for (annotation_guid,) in (
+            Annotation.query.join(Annotation.encounter)
+            .join(Encounter.sighting)
+            .filter(Sighting.guid == sighting_guid)
+            .values(Annotation.guid)
+        ):
+            annot = Annotation.query.get(annotation_guid)
+
+            for config_id in range(len(self.id_configs)):
+                conf = self.id_configs[config_id]
+                Annotation.query.get(annotation_guid)
+                matching_set_query = conf.get('matching_set')
+                # load=False should get us this response quickly, cuz we just want a count
+                matching_set = annot.get_matching_set(matching_set_query, load=False)
+                if not matching_set:
+                    log.info(
+                        f'Sighting {self.guid} send_all_identification annot {annot} {config_id} no matching set'
+                    )
+                    continue
+                for algorithm_id in range(len(conf['algorithms'])):
+                    if self._has_active_jobs(config_id, algorithm_id, str(annot.guid)):
+                        log.info(
+                            f'Sighting {self.guid} send_all_identification annot {annot} {config_id}{algorithm_id} has active jobs'
+                        )
+                        continue
+
+                    num_jobs += 1
+
+                    self.send_identification(
+                        config_id, algorithm_id, annot.guid, annot.content_guid
+                    )
+
+        if num_jobs > 0:
+            log.info(
+                f'Starting Identification for Sighting:{self.guid} using {num_jobs} jobs'
+            )
+            from .tasks import send_all_identification
```

Comment:
I think this is a bug and could explain some of the oddities I've been seeing. I think this is a merge error and should not be here.